ArdentHQ/arkscan
# refactor: statistics transaction records

## Summary

https://app.clickup.com/t/86dqn9gef

To test:

- run `php artisan explorer:cache-blocks`
- run `php artisan explorer:cache-transactions`
- see on page







## Checklist
